com.kms.cura_server/src/com/kms/cura_server/UserAPI.java

```diff
@@ -19,10 +20,14 @@
 import com.kms.cura.entity.json.JsonToEntityConverter;
 import com.kms.cura.entity.user.DoctorUserEntity;
 import com.kms.cura.entity.user.PatientUserEntity;
+import com.kms.cura.entity.user.UserEntity;
 import com.kms.cura_server.resources.Strings;
 
 @Path("/user")
 public final class UserAPI {
+	private boolean statusKey = true;
+	private String STATUS_KEY = "statuskey";
+	private String MESSAGE= "message";
```

Comment:
MESSAGE and STATUS_KEY should be move down to com.kms.entity for re-usable between client and server